src/pkg/message/message.go

```diff
@@ -39,6 +39,9 @@ const (
 // NoProgress tracks whether spinner/progress bars show updates.
 var NoProgress bool
 
+// LogStream is the stream to write logs to.
+var LogStream io.Writer = os.Stderr
```

Comment:
```suggestion
// LogWriter is the stream to write logs to.
var LogWriter io.Writer = os.Stderr
```

(as well as all the other places <3)